src/firebase/utils/batchUpdateData.js

```diff
@@ -0,0 +1,57 @@
+import db from '../firebase';
+import { awardConsiderationEnum } from '../../constants';
+
+/*
+* This batchUpdateData method is written explicitly for handling
+* a batch transaction of updating performance and adjudications collection.
+* This can be refactored to handled batch updates and transactions if needed.
+*/
+const batchUpdateData = async (
+  collectionName,
+  docName,
+  data,
+  choreoAwardAction,
+  specialAwardAction,
+) => {
+  const performanceCollection = collectionName.match(/events\/\w{20}\/performances/)[0];
+  const { length } = performanceCollection;
+  const performanceDocName = collectionName.substring(length + 1, length + 21);
```

Comment:
What is encapsulated by the substring at length + 1 and length + 21. Is this a performanceDocName or performanceDocID then?

Reply:
adding documentation above L4-35